lms/templates/problem.html

```diff
@@ -85,7 +85,7 @@ <h3 class="hd hd-3 problem-header" id="${ short_id }-problem-title" aria-describ
             </form>
         % endif
       % endif
-      <div class="submission-feedback" id="submission_feedback_${short_id}">
+      <div class="submission-feedback ${'cta-enabled' if submit_disabled_cta else ''}" id="submission_feedback_${short_id}">
```

Comment:
Wait, this feels confusing - the classname `cta-enabled` if the cta is disabled?

Reply:
The CTA is enabled. The `submit_disabled_cta` is saying the Submit button is Disabled so show the CTA.